Cargo.toml

```diff
@@ -108,6 +108,7 @@ new_debug_unreachable = "1.0.4"
 once_cell = "1.13.0"
 av1-grain = { version = "0.2.0", features = ["serialize"] }
 serde-big-array = { version = "0.4.1", optional = true }
+wide = "0.7.5"
```

Comment:
We can drop this new dependency for now, as it is unused.

Reply:
Don't worry, I'll pull it back in shortly 🙂